psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -107,36 +107,41 @@ def run_sapt_dft(name, **kwargs):
     core.IO.set_default_namespace('dimer')
     data = {}
 
-    if (core.get_global_option('SCF_TYPE') == 'DF'):
-        # core.set_global_option('DF_INTS_IO', 'LOAD')
+    if (core.get_global_option('SCF_TYPE') in ['DF', 'DISK_DF']):
+        # Save integrals
+        # We want to try to re-use itegrals for the dimer and monomer SCF's.
+        # If we are using Disk based DF (DISK_DF) then we can use the
+        # DF_INTS_IO option.  MemDF does not know about this option but setting
+        # it will be harmless there.
         core.set_global_option('DF_INTS_IO', 'SAVE')
```

Comment:
Can you double check that setting this does not change the DF algorithm to DiskDF? In the options docs for `DF_INTS_IO` it says `IO caching for CP corrections, etc. Changing this selects Disk_DF over Mem_DF. Note that setting this forces DiskDFJK when SCF_TYPE=DF.`

(see https://github.com/psi4/psi4/blob/master/psi4/src/read_options.cc#L1674)